src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/ObjectSynchronizer.java

```diff
@@ -93,47 +79,31 @@ public static Iterator objectMonitorIterator() {
 
   private static class ObjectMonitorIterator implements Iterator {
 
-    // JVMTI raw monitors are not pointed by gBlockList
-    // and are not included by this Iterator. May add them later.
```

Comment:
Are JVMTI raw monitors now in the `inUseList`?

Reply:
inUseList only contains ObjectMonitors. No raw monitors (JVM/TI or otherwise)
are managed on that list.

Reply:
Ok, then I think this comment needs to be retained.

Reply:
I reverted comment, and update it.